### DISCUSSION TITLE
Missing word in readme.md ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tadashi-aikawa/obsidian-another-quick-switcher#235] [Q&A] [answered]
Hello,

Just above the [2. Header floating search in file / Header search in file](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher#2-header-floating-search-in-file--header-search-in-file) section.

![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/641b67f2-95c4-460d-9db7-4843f5dfaf8d)

**Comment by tadashi-aikawa** [comment]
Thank you, @Fred-Vatin :)

It seems to have been escaped unexpectedly, so I will correct it. 😇 

![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/9500018/185111ab-04d0-4516-ace3-fbca0135e8c4)

**Reply by tadashi-aikawa** [reply, marked as answer]
@Fred-Vatin 
Fixed 👍 
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/9500018/a7bc5b7d-05ee-4294-ac96-9ff1fd4c83c7)
